src/ui/PrintableUri/PrintableUri.ts

```diff
@@ -48,67 +50,65 @@ export class PrintableUri extends ResultLink {
-    let xmlDoc: XMLDocument = Utils.parseXml(parentsXml);
-    let parents = xmlDoc.getElementsByTagName('parent');
-    let tokens: HTMLElement[] = [];
-    let separators: HTMLElement[] = [];
+    const xmlDoc: XMLDocument = Utils.parseXml(parentsXml);
+    const parents = xmlDoc.getElementsByTagName('parent');
+    const tokens: HTMLElement[] = [];
+    const separators: HTMLElement[] = [];
     for (let i = 0; i < parents.length; i++) {
       if (i > 0) {
-        let seperator = this.buildSeparator();
+        const seperator = this.buildSeparator();
         separators.push(seperator);
         element.appendChild(seperator);
       }
-      let parent = <Element>parents.item(i);
-      let token = this.buildHtmlToken(parent.getAttribute('name'), parent.getAttribute('uri'));
+      const parent = <Element>parents.item(i);
+      const token = this.buildHtmlToken(parent.getAttribute('name'), parent.getAttribute('uri'));
       tokens.push(token);
 
       element.appendChild(token);
     }
   }
 
   public renderUri(element: HTMLElement, result?: IQueryResult) {
-    let parentsXml = Utils.getFieldValue(result, 'parents');
+    const parentsXml = Utils.getFieldValue(result, 'parents');
     if (parentsXml) {
       this.renderParentsXml(element, parentsXml);
     } else {
       if (!this.options.titleTemplate) {
         this.uri = result.clickUri;
         let stringAndHoles: StringAndHoles;
         if (result.printableUri.indexOf('\\') == -1) {
-          stringAndHoles = StringAndHoles.shortenUri(result.printableUri, $$(element).width() / 7);
+          stringAndHoles = StringAndHoles.shortenUri(result.printableUri, $$(element).width());
         } else {
-          stringAndHoles = StringAndHoles.shortenPath(result.printableUri, $$(element).width() / 7);
+          stringAndHoles = StringAndHoles.shortenPath(result.printableUri, $$(element).width());
         }
         this.shortenedUri = HighlightUtils.highlightString(stringAndHoles.value, result.printableUriHighlights, stringAndHoles.holes, 'coveo-highlight');
-        let link = $$('div');
+        const link = $$('div');
         link.setAttribute('title', result.printableUri);
         link.addClass('coveo-printable-uri-part');
         link.setHtml(this.shortenedUri);
         link.setAttribute('href', result.clickUri);
         element.appendChild(link.el);
       } else if (this.options.titleTemplate) {
-        let newTitle = this.parseStringTemplate(this.options.titleTemplate);
+        const newTitle = this.parseStringTemplate(this.options.titleTemplate);
         this.element.innerHTML = newTitle ? StreamHighlightUtils.highlightStreamText(newTitle, this.result.termsToHighlight, this.result.phrasesToHighlight) : this.result.clickUri;
       }
     }
     element.title = this.result.printableUri;
   }
 
   public buildSeparator() {
-    let seperator = document.createElement('span');
-    seperator.innerText = ' > ';
+    const seperator = document.createElement('span');
```

Comment:
This should use the short hand method also with $$('span', {className : 'coveo-rpintable-uri-separator'}, ' > ');